shared/src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -952,7 +952,7 @@ namespace datadog::shared::nativeloader
         IUnknown* tstVerProfilerInfo;
         if (S_OK == corProfilerInfoUnk->QueryInterface(__uuidof(ICorProfilerInfo11), (void**) &tstVerProfilerInfo))
         {
-            Log::Info("ICorProfilerInfo11 available. Profiling API compatibility: .NET Core 5.0 or later.");
+            Log::Info("ICorProfilerInfo11 available. Profiling API compatibility: .NET Core 3.1 or later.");
```

Comment:
Can you also add checks for `ICorProfilerInfo12` (.NET 5) and `ICorProfilerInfo13` (.NET 7)?

Reply:
I don't know, let's find out